pkg/crd/crd.go

```diff
@@ -90,6 +88,12 @@ func NewCRDClientLite(config *rest.Config, nodeName string, scale, isStandardMod
 		return nil, err
 	}
 
+	idListInformer, err := newIDInformerLite(newIDListWatch(restClient))
```

Comment:
We don't need a new informer for `AzureIdentity` in the standard mode `AzureAssignedIdentity` for the pod contains the `AzureIdentity` reference. We can lookup the clientID and resourceID from all the `AzureAssignedIdentities` that match for the pod.

The only reason we have the informer for managed mode is, there'll be no MIC in managed mode to do the assignment.  The Identity assignment will be done by RP or externally by running `az vm/vmss identity assign` command. In this mode, NMI will lookup `AzureIdentity by first matching labels on `AzureIdentityBinding` and the `Pod`.